saltstack/base/salt/prerequisites/usr/bin/ccmv2-helper.sh

```diff
@@ -7,6 +7,7 @@ setup_ccmv2() {
   BACKEND_ID="${CCM_V2_AGENT_BACKEND_ID_PREFIX}${INSTANCE_ID}"
   BACKEND_HOST="localhost"
   BACKEND_PORT="9443"
+  ACC_KEY_TMP_FILE=$1
```

Comment:
I think the path can be hardwired here as `/etc/ccmv2/something`, no need to get it from a script argument. (Not because it hurts, but fixed paths are used elsewhere below like for `AGENT_KEY_PATH`, `AGENT_CERT_PATH` etc.)